features/main.feature

```diff
@@ -13,8 +13,9 @@ Feature: ProtonMail main functionality
         Then I expect Letter is filled properly with "testinaccountforjspurposes@protonmail.com" and "Test letter" fields
 
     Scenario: 3. Drafted letter is present inside Drafts folder
-        When I wait some time
+        When I wait "5" seconds
         Then I expect Letter is saved automatically
+        And I expect date has valid format
```

Comment:
you can define what format is valid in the case,
something like I expect date has dd.mm.yy format